spec/danger_plugin_spec.rb

```diff
@@ -316,6 +318,27 @@ module Danger
           expect(status[:warnings]).to_not be_empty
           expect(status[:errors]).to be_empty
         end
+
+        it 'parses environment variables set within the swiftlint config' do
+          ENV['ENVIRONMENT_EXAMPLE'] = 'excluded_dir'
```

Comment:
I took a look to see if there were a better way to stub out this environment variable, but [apparently there isn't](https://robots.thoughtbot.com/testing-and-environment-variables)! However, I would move `ENV['ENVIRONMENT_EXAMPLE'] = nil` into an `after(:each) do ... end` block, for two reasons:

- further tests with this variable shouldn't have to remember to reset it, and
- if this test raises and error halfway through, we want Rspec to reset the variable for us so the environment is clean for the next test.

Reply:
Good one!